core/controllers/dashboard.py

```diff
@@ -227,6 +231,8 @@ def _get_intro_card_color(category):
         self.values.update({
             'explorations_list': explorations_list,
             'collections_list': collections_list,
+            'weekly_dashboard_stats': user_services.get_user_dashboard_stats(
```

Comment:
get_user_dashboard_stats() at the moment doesn't seem to have any 'weekly' component. Is this misnamed?